cmd/api/src/docs/json/definitions/v2.json

```diff
@@ -951,6 +951,9 @@
             },
             "name": {
                 "type": "string"
+            },
+            "description": {
```

Comment:
The OAPI docs don't seem to be generating properly for any request that reference an external object. Those with embedded parameters are working, at least when testing locally.

Regardless, I updated docs here as well as in the new `.yaml` Redocly docs we will be moving to